db/migrate/20200524080344_create_storage_systems.rb

```diff
@@ -0,0 +1,17 @@
+class CreateStorageSystems < ActiveRecord::Migration[5.1]
+  def change
+    create_table :storage_systems do |t|
+      t.string :management_ip
+      t.string :name
+      t.string :secondary_ip
```

Comment:
This looks like it should be handled by `has_many :endpoints`, what happens if a system has a third IP addr?

Reply:
The fields are here because we are syncing them from the provider. 
But I'm not sure that we actually need them. I don't think we intend to access the systems directly from MIQ.
@ereztheodorou, what do you think?

Reply:
MIQ will not currently use them as they are managed by the provider. They are here for information only. in the future we might want to be able to update these IPs in the provider though MIQ interface (via the REST api) We can drop it for now.